packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -77,7 +83,8 @@ const Select = <T extends OptionValueType>({
   errorBorder,
   searchDisabled,
   inputContainerClassName,
-  endComponent
+  endComponent,
+  customMenuContent
```

Comment:
Input props are changed:
Few props such as: `inputClassName`, `labelClassName`, `inputContainerClassName` are removed from the Input component.

Reply:
Ok from all of the comments, this actually is the challenging one because these are the only way I found to make styling modifications from the parent component without internally affecting the Select component, as I mentioned I choose the approach that wouldn't break other select component implementations across all apps (admin, dashboard and studio) which is the main goal I aimed here, to avoid the intrinsic risk of breaking stuff.

I guess the question again is, are these UI component refactors expected for 0.9 or for a later release?

What we need to keep in mind is that we can't make breaking changes over components that already are implemented for the given release or we'll end up breaking stuff that already works for such release version, because as I said the intrinsic risk is too high, we'll end up breaking a lot of stuff, the thing is that I have no idea if the refactor of these core ui components are for 0.9 or not.

Reply:
https://github.com/ir-engine/ir-engine/pull/538#discussion_r1833708693